features/overlijden-indicatie-overleden.feature

```diff
@@ -0,0 +1,145 @@
+
+    Scenario: overleden persoon wordt geraadpleegd met burgerservicenummer, leeftijd wordt gevraagd
+      Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'overlijden' gegevens
+      | datum overlijden (08.10) | plaats overlijden (08.20) | land overlijden (08.30) |
+      | 20020701                 | 0518                      | 6030                    |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000152                       |
+      | fields              | leeftijd                        |
+      Dan heeft de response een persoon met alleen de volgende 'overlijden' gegevens
+      | naam               | waarde |
+      | indicatieOverleden | true   |
+
+  Rule: indicatieOverleden is in onderzoek wanneer ten minste één van de andere overlijden gegevens in onderzoek zijn
+
+    Abstract Scenario: <gegeven in onderzoek> is in onderzoek en overlijden velden worden niet gevraagd
+      Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'overlijden' gegevens
+      | naam                            | waarde                    |
+      | aanduiding in onderzoek (83.10) | <gba in onderzoek waarde> |
+      | datum ingang onderzoek (83.20)  | 20020701                  |
+      | datum overlijden (08.10)        | 20020701                  |
+      | plaats overlijden (08.20)       | 0518                      |
+      | land overlijden (08.30)         | 6030                      |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000152                       |
+      | fields              | burgerservicenummer             |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                                                    | waarde      |
+      | burgerservicenummer                                     | 000000152   |
+      | overlijden.indicatieOverleden                           | true        |
+      | overlijden.inOnderzoek.indicatieOverleden               | true        |
+      | overlijden.inOnderzoek.datumIngangOnderzoek.type        | Datum       |
+      | overlijden.inOnderzoek.datumIngangOnderzoek.datum       | 2002-07-01  |
+      | overlijden.inOnderzoek.datumIngangOnderzoek.langFormaat | 1 juli 2002 |
+
+      Voorbeelden:
+      | gegeven in onderzoek | gba in onderzoek waarde |
+      | categorie overlijden | 060000                  |
+      | groep overlijden     | 060800                  |
+      | datum overlijden     | 060810                  |
+      | plaats overlijden    | 060820                  |
+      | land overlijden      | 060830                  |
```

Comment:
Plaats en land kunnen hier denk ik uit. Als plaats en/of land in onderzoek staat, is indicatie overleden niet in onderzoek.